Makefile

```diff
@@ -22,7 +22,7 @@ get-openapi-sandbox:
 generate-api:
 	rm -rf src/api && \
     mkdir src/api && \
-	docker run --rm -v $(shell pwd):/app openapitools/openapi-generator-cli generate \
+	docker run --rm -v $(shell pwd):/app openapitools/openapi-generator-cli:v6.2.1 generate \
```

Comment:
This version needs to be fixed as the latest version results in changes to the generated api configuration